modules/ROOT/pages/installation/kubernetes/helm-charts.adoc

```diff
@@ -110,3 +183,35 @@ tolerations: []
 
 affinity: {}
 ----
+
+=== Using pre-configured secrets
+
+Adding senstive information as plain text in `values.yaml` is less secure in some environments. Such environments would have secrets being configured externally by privileged
+users or secure service agents such as link:https://developer.hashicorp.com/vault/tutorials/kubernetes/kubernetes-sidecar[Hashicorp Vault agent]. These securely pre-configured
+secrets can be used to set sensitive values for NOM server helm chart using `secretsFromSecrets` configuration. This value requires a `secretName` and a `key` for a NOM secret
+value. Following is an example values snippet that demonstrates this usecase with inline comments:
+
+[source,yaml]
+----
+secretsFromSecrets:
+  storageUri: # More than one secrets can be mapped to different values of NOM secrets
```

Comment:
I don't understand the comment here.

Reply:
yeah it may confuse more than clarify, let me add the example values instead which will make this point clear